packages/esm-service-queues-app/src/queue-table/queue-table.component.tsx

```diff
@@ -76,8 +76,10 @@ function QueueTable({ queueEntries, queueTableColumns, ExpandedRow, tableFilter
               <TableHead>
                 <TableRow>
                   {ExpandedRow && <TableExpandHeader enableToggle {...getExpandHeaderProps()} />}
-                  {headers.map((header) => (
-                    <TableHeader {...getHeaderProps({ header })}>{header.header}</TableHeader>
+                  {headers.map((header: QueueTableColumn) => (
+                    <TableHeader {...getHeaderProps({ header })}>
+                      <header.HeaderComponent />
```

Comment:
So I know you created the HeaderComponent based on my previous feedback regarding translation, but what isn't clear is why you didn't just continue to do the translation here when the column is rendered? i.e. what was wrong with just having an i18nKey like you had before, and then running it through the translation system here?

Reply:
The translation pre-commit git hook looks at places where we call the `t` function with hard-coded strings to determine what keys to add/remove in the translation json files. It does not have the ability to recognize those keys when they were passed into the `t` function like we do currently.

We got away with doing that because the `<ActiveVisitsTable>` hard-coded most of the same translation keys anyway, but with that gone, we have to do it differently.